README.md

```diff
@@ -57,7 +60,9 @@ The following attributes are available for `viam:camera:filtered-camera` bases:
 | `camera` | string | **Required** | The name of the camera to filter images for. |
 | `vision_services` | list | **Required** | A list of 1 or more vision services used for image classifications or detections. |
 | `vision` | string | **Required** | \*\***DEPRECATED**\*\* The vision service used for image classifications or detections. |
-| `window_seconds` | float64 | Optional | The size of the time window (in seconds) during which images are buffered. When a condition is met, a confidence score for a detection/classification exceeds the required confidence score, the buffered images are stored, allowing us to see the photos taken in the N number of seconds preceding the condition being met. |
+| `window_seconds` | float64 | Optional | The size of the time window (in seconds) during which images are buffered. When a condition is met, a confidence score for a detection/classification exceeds the required confidence score, the buffered images are stored, allowing us to see the photos taken in the N number of seconds preceding the condition being met. If this value is set, the 'window_seconds_before' and 'window_seconds_after' variable will be set to 0.0, even if you choose to set them in the config. |
+| `window_seconds_before` | float64 | Optional | The size of the time window (in seconds) before the condition is met, during which images are buffered. This allows you to see the photos taken in the N number of seconds preceding the condition being met. |
```

Comment:
I don't like the mention of N because it's not defined anywhere, though I get the gist of what you meant and have trouble suggesting something better. Maybe "store images from this many seconds before the condition is met" or similar? Same suggestion on the next line, too.